### DISCUSSION TITLE
No Test Runner in latest version of keystone-6

### DISCUSSION THREAD

**Discussion by sharmsa66** [keystonejs/keystone#8251] [Questions] [answered]
1. install latest version of keystone
2. check for test runner import { setupTestEnv, TestEnv } from '@keystone-6/core/testing'; import { setupTestRunner } from "@keystone-6/core/testing";
3. No test runner in the module

We should get  setupTestRunner  as it is helpful for unit testing

**Comment by flexdinesh** [comment, marked as answer]
The exports in `@keystone-6/core/testing` have been removed and replaced with `resetDatabase`. It's a breaking change in [core@v3](https://github.com/keystonejs/keystone/releases/tag/2022-10-19). Please check PR #7968 on how to use the new API in your tests.

FYI, our testing guide is out of date and needs to be updated and we're tracking it here #8169.
